homeassistant/components/roomba/__init__.py

```diff
@@ -1 +1,173 @@
 """The roomba component."""
+import asyncio
+import logging
+
+import async_timeout
+from roomba import Roomba, RoombaConnectionError
+import voluptuous as vol
+
+from homeassistant import config_entries, exceptions
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+
+from .const import (
+    BLID,
+    COMPONENTS,
+    CONF_BLID,
+    CONF_CERT,
+    CONF_CONTINUOUS,
+    CONF_DELAY,
+    CONF_NAME,
+    DEFAULT_CERT,
+    DEFAULT_CONTINUOUS,
+    DEFAULT_DELAY,
+    DOMAIN,
+    ROOMBA_SESSION,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_HOST): str,
+                vol.Required(CONF_USERNAME): str,
```

Comment:
@Cyr-ius Since this is going to be a breaking change and users have to set this up again, lets use `CONF_BLID` here instead of `CONF_USERNAME` so we don't have to maintain both

Reply:
I don't mind, but I thought you didn't want a change.
I made the change and I also deleted the conversion of the variable in the config_flow for the import

Reply:
I didn't realize this was a breaking change at the time so I was trying to avoid that.